aw-watcher-bash-preexec.sh

```diff
@@ -0,0 +1,35 @@
+export PATH=$PATH:~/.local/bin/
+
+CURRENT_SHELL=$(ps -p "$$" | tail -n 1 | awk '{ print $4 }')
+if [[ $CURRENT_SHELL == "zsh" ]]; then
+  # zsh supports preexec natively, do nothing
+  : # : is a nop
+elif [[ -f ~/.bash-preexec.sh ]]; then
+  source ~/.bash-preexec.sh
+else
+  echo "preexec is not supported by your terminal and bash-preexec is not installed at ~/.bash-preexec.sh, aw-watcher-bash will not work"
+fi
+
+send_aw_watcher_bash_event() {
+  local base_args=("$PPID" 'bash' "$(date --iso-8601=ns)")
```

Comment:
Exchange `'bash'` with the current_shell variable

Reply:
fixed